jni/lib/src/lang/jstring.dart

```diff
@@ -59,10 +59,7 @@ class JString extends JObject {
   /// after conversion and this object will be marked as released.
   String toDartString({bool releaseOriginal = false}) {
     ensureNotNull();
-    final length = Jni.env.GetStringLength(reference);
-    final chars = Jni.env.GetStringChars(reference, nullptr);
-    final result = chars.cast<Utf16>().toDartString(length: length);
```

Comment:
With this cast removed, `package:ffi` is no longer needed. Please remove the import to make the analyzer happy!

```
warning • Unused import: 'package:ffi/ffi.dart' • lib/src/lang/jstring.dart:7:8 • unused_import
```